config/locales/ui.en-GB.yml

```diff
@@ -442,14 +417,15 @@ en-GB:
             <p class="secondary">You can’t change your petition after this point.</p>
             <p class="secondary">This is a good time to check that your petition is clear. If you’ve included personal information, such as details about your health, your family or your job, think carefully about whether you are happy for this to be published.</p>
             <p class="secondary">Published petitions can only be removed from the site with the agreement of the Petitions Committee.</p>
-        replay_closing_date_stage:
-          heading: "Check your closing date"
-          custom_message_html: |-
-            <h1><span class="heading-secondary">Deadline</span>%{date}</h1>
-            <p>Your petition “%{action}” will be closed at midnight on this date. If it has received %{num} or more signatures then it will be referred to the Petitions Committee.</p>
-          default_message_html: |-
-            <h1><span class="heading-secondary">Deadline</span>Six months from publication</h1>
-            <p>Your petition “%{action}” will be closed at midnight on this date. If it has received %{num} or more signatures then it will be referred to the Petitions Committee.</p>
+        collect_signatures_stage:
+          heading: Do you want to collect signatures for your petition?
+          help_html: "<p>You can either write a letter to your parliament (because that's what a petition without signatures is, really) or collect signatures. Choose an option below.</p>"
+          label:
+            false: I want to lodge my petition immediately
+            true: I want to collect signatures
+          hint:
+            false: Just a letter then
+            true: Your petition will collect signatures for 4 weeks
           submit_html: |-
             <button type="submit" name="move_next" class="button forward" data-disable-with="This looks good">This looks good</button>
```

Comment:
I'd change this button text to 'Continue' - 'This looks good' doesn't really make sense on non-preview page. You'll need to change your cucumber steps as well.